ui/mirage/handlers/oidc-config.js

```diff
@@ -1,4 +1,5 @@
 export default function (server) {
+  server.post('/sys/capabilities-self', () => {});
```

Comment:
This will break any oidc config permissions tests that use this mirage setup. Just something to be aware of. Can we add it in the test after initiating this scenario?

Reply:
good to know! I've been using the helper `overrideMirageResponse` to do just that, override a response and return a 403, which has worked so far to test permissions - but I'll make a note for when I'm working on the acceptance test